src/main/java/use_cases/GameCreationRequestModel.java

```diff
@@ -0,0 +1,39 @@
+package use_cases;
+
+import java.util.HashMap;
+
+/**
+ * A request model containing User provided information on the Players to be made in a Game.
+ */
+public class GameCreationRequestModel {
+    private HashMap<String, Boolean> playersInfo;
+
+    /**
+     * Construct a GameCreationRequestModel
+     * @param info A HashMap where keys are potential Player names and keys are booleans which are true iff the
+     *             Player is to be a ComputerPlayer.
+     */
+    public GameCreationRequestModel(HashMap<String, Boolean> info) {
+        this.playersInfo = info;
+    }
+
+    /**
+     * Given a Player name, get whether this Player is to be a ComputerPlayer.
+     * @param playerName The name of the Player to check
+     * @return True iff the Player is to be a ComputerPlayer, null iff the name is not of a Player to be.
+     */
+    public Boolean isComputer(String playerName) {
```

Comment:
isComputerPlayer might be a better name to illustrate the purpose of this method.